src/configs/gemm-config.c

```diff
@@ -967,6 +967,30 @@ static void init_f32_gemm_config(void) {
       f32_gemm_config.mr = 4;
       f32_gemm_config.nr = 4;
     }
+  #elif XNN_ARCH_RISCV && XNN_ENABLE_RISCV_VECTOR
+    const struct xnn_hardware_config* hardware_config = xnn_init_hardware_config();
+    assert(hardware_config != NULL);
+    if (hardware_config->use_riscv_vector) {
+      f32_gemm_config.linear.gemm[XNN_MR_TO_INDEX(7)] = xnn_init_hmp_gemm_ukernel((xnn_gemm_ukernel_fn) xnn_f32_gemm_ukernel_7x4v__rvv);
+      f32_gemm_config.linear.gemm[XNN_MR_TO_INDEX(1)] = xnn_init_hmp_gemm_ukernel((xnn_gemm_ukernel_fn) xnn_f32_gemm_ukernel_1x4v__rvv);
```

Comment:
although faster, we dont normally do linear except on wasmsimd where minmax is very slow.
Its okay, but increases the code size (3 kernels instead of 1)

Reply:
oh... I finally got what you mentioned...